### DISCUSSION TITLE
Triggering automations doesn't work since tvOS 15.

### DISCUSSION THREAD

**Discussion by pax0707** [AlexGustafsson/homebridge-wol#182] [Questions]
I've been using the power state to trigger automation for turning on/off mood lights for my TV and PC but since updating to 15 automation stopped running. 
Could it be the issue with the plugin or is it something 15 related?

It was suggested that it could be related to a missing push flag to that on/off switch so it doesn't work in automation as a trigger.

**Comment by AlexGustafsson** [comment]
> It was suggested that it could be related to a missing push flag to that on/off switch so it doesn't work in automation as a trigger.

That could be it. I don't recall handling any push flag.
